cmake/GtwrapUtils.cmake

```diff
@@ -103,3 +103,37 @@ macro(gtwrap_get_python_version)
   configure_python_variables()
 
 endmacro()
+
+# Concatenate multiple wrapper interface headers into one.
+# The concatenation will be (re)performed if and only if any interface files
+# change.
+#
+# Arguments:
+# ~~~
+# destination: The concatenated master interface header file will be placed here.
+# inputs (optional): All the input interface header files
+function(combine_interface_headers
```

Comment:
What's up with this? Not part of this PR.

Reply:
my bad... temporarily i've just changed the base of this PR, but if this goes through before the other PR then I'll make a new branch/PR to merge